stytch-react-dotnet-example.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This seems like something that should be in the .gitignore as well `*.sln`

Reply:
adding!